sha1/src/lib.rs

```diff
@@ -26,12 +27,16 @@ use digest::const_oid::{AssociatedOid, ObjectIdentifier};
 #[cfg(feature = "zeroize")]
 use digest::zeroize::{Zeroize, ZeroizeOnDrop};
 
+#[cfg(feature = "std")]
+extern crate std;
```

Comment:
I have no idea why I need this, but I am unable to run standard tests without this, am I doing sth wrong?

Reply:
You forgot to use `default-features = false` when you added `sha1` dependency, since `std` is a default feature, it gets enabled.

Reply:
when I remove it from both `sha1` and `sha1-checked` I get this

```
cargo test -p sha1-checked
   Compiling sha1 v0.11.0-pre.3 (/Users/dignifiedquire/opensource/rustcrypto/hashes/sha1)
   Compiling sha1-checked v0.11.0-pre.3 (/Users/dignifiedquire/opensource/rustcrypto/hashes/sha1-checked)
error[E0433]: failed to resolve: use of undeclared crate or module `std`
   --> sha1-checked/src/lib.rs:257:6
    |
257 | impl std::io::Write for Sha1 {
    |      ^^^ use of undeclared crate or module `std`
```